### PR TITLE
EMD Documentation

### PR DESCRIPTION
**Pull request by marci73**

### CODE REVIEW

File: docs/methods/properties/x-ray/index.md [hunk 1]
```diff
@@ -9,5 +9,4 @@ delta-scf
 tddft
 delta-kick
 correction_scheme
-resonant_excitation
```
Comment (oschuett) [inline docs/methods/properties/x-ray/index.md:12]: Did you mean to rename the page? Every page has to be listed in the index or Sphinx will be unhappy.

File: docs/methods/sampling/ehrenfest.md [hunk 2]
```diff
@@ -1,3 +1,523 @@
...
+(`STEPS`). It is crucial to set an appropriate time step to propagate the electronic degrees of
+freedom, i.e., in the order of atto-seconds. All other input parameters related to the RT-TDDFT run
+are specified from the section
+[REAL_TIME_PROPAGATION](#CP2K_INPUT.FORCE_EVAL.DFT.REAL_TIME_PROPAGATION). The simulation needs to
+start from the electronic density at $t_0$ (`INITIAL_WFN`). This can be the ground state obtained by
+means of an initial SCF optimization (`SCF_WFN`) or providing the restart file of a previously
+computed state (`RESTART_WFN`). The propagation can also be restarted setting the initial
+wavefunction as `RT_RESTART` and providing the correct restart file, which has a different format
+with respect to the SCF restart file. The path to the restart file is in both cases provided by the
+usual [RESTART_FILE_NAME](#CP2K_INPUT.FORCE_EVAL.DFT.RESTART_FILE_NAME) key.
```
Comment (oschuett) [inline docs/methods/sampling/ehrenfest.md]: Aliases don't work, you have to refer to keywords by their primary name.
In this case it's `#CP2K_INPUT.FORCE_EVAL.DFT.WFN_RESTART_FILE_NAME`

File: docs/methods/properties/x-ray/index.md (hunk 1 above)
Comment (marci73) [reply docs/methods/properties/x-ray/index.md:12]: I want to remove that resonant_excitation page, that is horribly written and also partially wrong
anyway not in the right place

File: docs/methods/sampling/ehrenfest.md (hunk 2 above)
Comment (marci73) [reply docs/methods/sampling/ehrenfest.md]: Is it not possible just to have it written with different font, without linking the keyword in the input?

Comment (oschuett) [reply docs/methods/sampling/ehrenfest.md]: You could format it as code by putting it in blockquotes:

```
`RESTART_FILE_NAME`
```

Or you could keep it as a link like this:
```
[RESTART_FILE_NAME](#CP2K_INPUT.FORCE_EVAL.DFT.WFN_RESTART_FILE_NAME)
```

File: docs/methods/properties/x-ray/index.md (hunk 1 above)
Comment (oschuett) [reply docs/methods/properties/x-ray/index.md:12]: Sphinx does not like unreachable files. So if you remove the page from the index then you also have to delete the `resonant_excitation.md` file.

File: docs/methods/sampling/ehrenfest.md (hunk 2 above)
Comment (marci73) [reply docs/methods/sampling/ehrenfest.md]: I think I did exactly that,  but there was a part of the name missing

File: docs/methods/properties/x-ray/index.md (hunk 1 above)
Comment (marci73) [reply docs/methods/properties/x-ray/index.md:12]: Ah OK

**Pull request merged**